### ISSUE TITLE
Permission denied with RC.83 and Docker

### ISSUE THREAD

**Issue by pierre-H** [directus/directus#6574]
### Describe the Bug

Can't launch Directus RC.83 with Docker compose.
No problem with RC.82

This is the Docker output : 

```shell
npm WARN exec The following package was not found and will be installed: directus

npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated

npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated

npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.

npm WARN deprecated har-validator@5.1.5: this library is no longer supported

npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.

npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.

npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142

npm WARN deprecated node-pre-gyp@0.11.0: Please upgrade to @mapbox/node-pre-gyp: the non-scoped node-pre-gyp package is deprecated and only the @mapbox scoped package will recieve updates in the future

npm WARN deprecated querystring@0.2.0: The

npm WARN deprecated uuid@3.3.2: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.

sh: directus: Permission denied

npm notice 
npm notice New minor version of npm available! 7.15.1 -> 7.19.0
npm notice Changelog: <https://github.com/npm/cli/releases/tag/v7.19.0>
npm notice Run `npm install -g npm@7.19.0` to update!
npm notice 

```

### To Reproduce

Docker composer file :

```yaml
    api:
        env_file:
            - "./.directus.env"
        image: directus/directus:v9.0.0-rc.83
        ports:
            - 3103:8055
        user: "${UID}:${GID}"
        volumes:
            - ./data2/uploads:/directus/uploads
            - ./data2/extensions:/directus/extensions
```

### What version of Directus are you using?

RC.83

### What version of Node.js are you using?

The one provided by the docker container

### What database are you using?

Mariadb  10.5.11

### What browser are you using?

Firefox

### What operating system are you using?

Windows

### How are you deploying Directus?

Docker Compose V2

**Comment by nazariydj** [comment]
I have same warnings but the app starts... for now l'm downgrade to v9.0.0-rc.82

**Comment by wc-matteo** [comment]
![image](https://user-images.githubusercontent.com/10562635/123992417-e34d1400-d9cb-11eb-8920-076c69401222.png)

on k8s

**Comment by t7tran** [comment]
Nope, I just raised this on Discord /docker.

According to the below logic in the `Dockerfile`...

# (retry if it fails for some reason, e.g. release not published yet)
  for i in $(seq 10); do npm install "directus@${VERSION}" && break || sleep 30; done

if the package wasn't published in time (after 5 mins), it would continue without the dependencies installed.

In my observation, it should continue to start up. However, it may depend on the environment. On Kubernetes, if the pod doesn't have internet access, this would fail, then it becomes complicated. `502` error means that the pod is running (at least Kubernetes assumes so). `503` error is expected in this case. I assume this K8s deployment doesn't have a readiness probe.

In short, we just need to fix that `npm install "directus@${VERSION}"` logic first. Then things should fall back into place.

**Comment by t7tran** [comment]
Oh, I missed `sh: directus: Permission denied` line. As he could run it with rc.82, I don't think it's related.
